apps/web/src/app/(authenticated)/insights/components/insight-card.tsx

```diff
@@ -35,50 +33,21 @@ interface Datapoint {
 export default function InsightsGrid(props: InsightCardProps): ReactNode {
   const format = useFormatter();
   const t = useTranslations('insights');
-  // const computedColorScheme = useComputedColorScheme();
-  // const theme = useMantineTheme();
   const [rank, setRank] = useState<RankType>({
-    label: '-',
-    color: 'gray',
+    label: 'GOOD',
+    color: 'green',
   });
   const [datapoints, setDatapoints] = useState<Datapoint[]>([]);
 
-  // Set the correct color of the icons based on the current color scheme
-  // let iconColor = theme.colors.dark[2];
-  // if (computedColorScheme === 'dark') {
-  //   iconColor = theme.colors.gray[5];
-  // }
-
   const setupRank = useCallback(() => {
-    switch (props.rank) {
-      case 'good':
-        setRank({
-          label: 'GOOD',
-          color: 'green',
-        });
-        break;
-      case 'mid':
-        setRank({
-          label: 'CAUTION',
-          color: 'yellow',
-        });
-        break;
-      case 'bad':
-        setRank({
-          label: 'BAD',
-          color: 'red',
-        });
-        break;
-      case 'unknown':
-        setRank({
-          label: '-',
-          color: 'gray',
-        });
-        break;
-      default:
-        break;
+    // If the latest CPM is lower than the previous CPM, rank is set to CAUTION
+    if (props.datapoints[props.datapoints.length - 1].cpm < props.datapoints[props.datapoints.length - 2].cpm) {
```

Comment:
```suggestion
    if (props.datapoints[props.datapoints.length - 1].cpm > props.datapoints[props.datapoints.length - 2].cpm) {
```
CPM is cost per thousands, which mean that less is better